doc/extensions/authoring.md

```diff
@@ -51,7 +51,7 @@ We detect Python package version via `python setup.py --version`. Only when the
 
 The storage fields can be stored in your config file or as environment variables so you need not supply them every time. Once the publish command has been run (you must be logged in to the Azure CLI for it to succeed), you can open a PR that will contain your code changes and the index update. This used to be done in two steps.
 
-Once your extension is published, you can view it via `az extension list-avaliable -o table`.
+Once your extension is published, you can view it via `az extension list-available -o table`.
```

Comment:
This typo was from https://github.com/Azure/azure-cli/pull/13960. It should definitely be found out by a text editor with spell check.

Reply:
I know sometimes spell checks aren't always caught if they're in code snippets, which may have been the case.